src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -310,10 +321,12 @@ std::string tenantFromDb(std::string& database)
 {
   std::string r;
   std::string prefix  = dbPrefix + "-";
+
   if (strncmp(prefix.c_str(), database.c_str(), strlen(prefix.c_str())) == 0)
   {
     char tenant[MAX_SERVICE_NAME_LEN];
```

Comment:
MAX_SERVICE_NAME_LEN is defined in MongoGlobal.h.
I just changed it from 1024 to 50.